src/main/webapp/index.html

```diff
@@ -48,10 +48,58 @@
           <button data-modal-close-button class="exit-button">&times;</button>
         </div>
         <div class="modal-body" id="results-body">
-          <div id="results-list-container"></div>
+          <div class="list" id="results-list-container"></div>
         </div>
         <div class="modal-body" id="reviews-body">
-          <div id="reviews-list-container"></div>
+          <div class="list" id="reviews-list-container"></div>
```

Comment:
Why make this of the list class? I'm unfamiliar with it.

Reply:
"list" is a made up name, I just needed to access two specific elements in js so I assigned them the same class. It made the js smoother.